lib/car.rb

```diff
@@ -0,0 +1,25 @@
+class Car
+  attr_reader :make, :model, :monthly_payment
```

Comment:
Since you made methods for `make` and `model` below you wouldn't add them to attr_reader. You only need to add attributes that don't have their own method. `monthly_payment` for example is good! You defined it in the initialize and then since you added it to attr_reader you can access it on a car. 

You should do the same thing for loan_length rather than creating a method for it like you are doing on line 17. 

```suggestion
  attr_reader :monthly_payment, :loan_length
```